### DISCUSSION TITLE
Change recharge costs?

### DISCUSSION THREAD

**Discussion by daviegeepers** [teslamate-org/teslamate#2167] [Q&A]
Hi. A lot of my charging costs are wrong. I didn't set the price per kWh for awhile so those costs are maker as several hundred dollars. 

Is there a way to set the cost per kWh ($0.10) and have it recalculate all the existing charges?

Right now it says I spent 7000.00 on charges and it should be like 90 dollars. 

Thanks. Sorry if I'm not making sense.

**Comment by boba51** [comment]
Still not sure how to reply not answer sorry :(

Looking for a similar option, I set the wrong rate for my home charges, it populated all 200+ of them, now I need to adjust, any way to remove/reset them so I can re-enter the correct rate, I am searching the forum, but haven't found anything yet.
